packages/router-plugin/tests/router.plugin.spec.ts

```diff
@@ -74,6 +74,39 @@ describe('NgxsRouterPlugin', () => {
       expect(routerState!.url).toEqual('/a-path');
     });
   }));
+
+  it('should select custom router state ', fakeAsync(async () => {
```

Comment:
Is the `async` really required?

Reply:
@eranshmil Regarding the [official documentation](https://ngxs.gitbook.io/ngxs/recipes/unit-testing) you need to wrap the test using Angular's `async` method. I tried without the `async` and it also works so maybe the `fakeAsync` method is also creating an asynchronous test zone? 🤔

Reply:
Yeah, I wrote that in the documentation. ;)
Since then, I've figured out that it's not mandatory and not all the time you need to wrap it with angular's async().

And aside from that, you used the async-await modifier, and not the angular's utility.

Reply:
haha that's embarassing 😅! I removed the `async` keyword since it's not necessary. I reused it from another test and didn't notice it was the modifier, my bad!

Reply:
Thanks, I'm trying to figure out this mystery for some time, when we need to use it.